parser.y

```diff
@@ -10771,4 +10788,134 @@ LoadStatsStmt:
 		}
 	}
 
+/********************************************************************************************
+ *
+ *  Create Sequence Statement
+ *
+ *  Example:
+ *      CREATE SEQUENCE IF NOT EXIST my_sequence
+ *          [INCREMENT BY <increment>]
+ *          [START WITH <start>]
+ *	    [MINVALUE <minvalue> | NOMINVALUE]
+ *          [MAXVALUE <maxvalue> | NOMAXVALUE]
+ *          [CACHE <cache>]
+ *          [CYCLE | NO CYCLE]
+ *          [ORDER | NO ORDER]
+ ********************************************************************************************/
+
+CreateSequenceStmt:
+	"CREATE" OrReplace OptTemporary "SEQUENCE" IfNotExists TableName CreateSequenceOptionListOpt CreateTableOptionListOpt
+	{
+		$$ = &ast.CreateSequenceStmt{
+			OrReplace: $2.(bool),
+			IsTemporary: $3.(bool),
+			IfNotExists: $5.(bool),
+			Name: $6.(*ast.TableName),
+			SeqOptions: $7.([]*ast.SequenceOption),
+			TblOptions: $8.([]*ast.TableOption),
+		}
+	}
+
+CreateSequenceOptionListOpt:
+	{
+		$$ = []*ast.SequenceOption{}
+	}
+|       SequenceOptionList
+
+SequenceOptionList:
+	SequenceOption
+	{
+		$$ = []*ast.SequenceOption{$1.(*ast.SequenceOption)}
+	}
+|	SequenceOptionList SequenceOption
+	{
+		$$ = append($1.([]*ast.SequenceOption), $2.(*ast.SequenceOption))
+	}
+
+SequenceOption:
+	"INCREMENT" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceOptionIncrementBy, IntValue: $2.(int64),}
+	}
+|	"INCREMENT" "=" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceOptionIncrementBy, IntValue: $3.(int64),}
+	}
+|	"INCREMENT" "BY" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceOptionIncrementBy, IntValue: $3.(int64),}
+	}
+|	"INCREMENT" "BY" "=" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceOptionIncrementBy, IntValue: $4.(int64),}
+	}
+|	"START" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceStartWith, IntValue: $2.(int64),}
+	}
+|	"START" "=" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceStartWith, IntValue: $3.(int64),}
+	}
```

Comment:
You could use `EqOpt` to merge these rules.

```
|	"START" EqOpt SignedNum
```

Reply:
Nice catch!